.github/workflows/testing.yaml

```
@@ -0,0 +1,66 @@
+name: testing
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+
+  run-tests:
```

Please, remove the newline here or add a newline to the section into check.yaml. Just to keep it in the same style.
```suggestion
jobs:
  run-tests:
```